egs2/opencpop/svs1/conf/tuning/train_visinger_hubert_cn_mert.yaml

```diff
@@ -0,0 +1,223 @@
+# This configuration is for ESPnet2 to train VITS, which
```

Comment:
The input fs is not reflected in the config, might be better to do that as an example to others